CHANGELOG.md

```diff
@@ -3,6 +3,23 @@
 	Placeholder for next release:
 	## __WORK IN PROGRESS__
 -->
+## __WORK IN PROGRESS__ · _Summer is coming!_
+### Breaking changes · [Migration guide](https://zwave-js.github.io/node-zwave-js/#/getting-started/migrating-to-v7)
+* Renamed `controller.removeNodeFromAllAssocations` to `controller.removeNodeFromAllAssociations` to fix a typo
+* We've reworked/fixed the parsing of Node Information Frames (NIF) to match the specifications and changed node properties to make more sense
```

Comment:
I also remember something about the `interview completed` event that isn't triggered anymore right?

Reply:
I'll start testing this very shortly.

Nice work 👍

Reply:
https://github.com/zwave-js/node-zwave-js/pull/2030/files?file-filters%5B%5D=.md#diff-0673db990a9385cee3ac29f1a4709dfeb8ad1cfe59fc9267ef4ad81b7de1ef13R5